css-three-column-layout/styles.css

```diff
@@ -0,0 +1,63 @@
+* {
+  background-color: #1b96da;
+}
+
+body {
+  color: white;
+}
+
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column {
```

Comment:
Why not be explicit and name it `column-full`?

Reply:
because of bad practice. fixed!